shotover-proxy/src/transforms/protect/mod.rs

```diff
@@ -269,7 +269,9 @@ mod protect_transform_tests {
     use crate::transforms::protect::ProtectConfig;
     use crate::transforms::{Transform, Transforms, Wrapper};
 
-    #[tokio::test(flavor = "multi_thread")]
+    //#[tokio::test(flavor = "multi_thread")]
+    #[allow(unused)]
+    #[ignore]
```

Comment:
Can you remove these changes please?
Dont want merge conflicts with https://github.com/shotover/shotover-proxy/pull/484

Reply:
I'll rebase now that the test is fixed.